scripts/sign_windows_executable.js

```diff
@@ -0,0 +1,169 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const minimist = require('minimist');
+const fsPromises = require('fs/promises');
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+
+function assert(condition, msg) {
```

Comment:
I feel we can abstract these functions into `utilities/assert.mjs`.

Reply:
maybe part of `tools`? I like utilities more but maybe we can just rename tools to utilities once we clean it out some more later

Reply:
this could also be done in another PR, as i think this assert function could definitely be incorporated into some of the other scripts